cve_bin_tool/checkers/gstreamer.py

```diff
@@ -11,16 +11,11 @@
 def guess_contains(lines):
     """Tries to determine if a file includes expat
     """
-    count = 0
-    should_contain = [
-        "http://bugzilla.gnome.org/enter_bug.cgi?product=GStreamer",
-        "LGPL",
-    ]
     for line in lines:
-        for i in should_contain:
-            if i in line:
-                count += 1
-    return True if count >= len(should_contain) else False
+        if "http://bugzilla.gnome.org/enter_bug.cgi?product=GStreamer" in line:
+            return True
+        if "LGPL" in line:
```

Comment:
I don't think this line is going to work -- there's just too many things other than gstreamer that are LGPL (it's a license, and a fairly popular one).  It wasn't so bad when we were looking for *both* strings to be available, but if just matching LGPL we're going to get a lot of false positives.

It might be fine to just match on the bugzilla line, or find something else moderately unique as a backup line and replace LGPL as a signature string.